addons/website/static/src/js/content/menu.js

```diff
@@ -222,4 +222,19 @@ publicWidget.registry.menuDirection = publicWidget.Widget.extend({
         $menu.addClass('dropdown-menu-' + alignment);
     },
 });
+
+publicWidget.registry.toggleMegaMenu = publicWidget.Widget.extend({
+    selector: 'a.o_mega_menu',
+    events: {
+        'click': '_onClick',
+    },
+
+    /**
+     * @override
+     */
+    _onClick: function (ev) {
+        var megaMenuID = $(ev.currentTarget).find('span').attr('data-oe-id');
+        $('span[data-oe-id="' + megaMenuID + '"').find('div.s_mega_menu').toggleClass('d-none');
+    },
+});
```

Comment:
Why do we need a widget for this ? Opening a dropdown menu is handled automatically by bootstrap